### PR TITLE
[CP][Impeller] disable all Adrenos older than 630 (#57069)

### PR DESCRIPTION
**Pull request by jonahwilliams**
Cherry pick for https://github.com/flutter/flutter/issues/160041

### CODE REVIEW

**Review by zanderso — APPROVED** [review]

**Comment by jonahwilliams** [comment]
This should be working now.

File: impeller/renderer/backend/vulkan/driver_info_vk_unittests.cc [hunk 1]
```diff
@@ -80,6 +80,18 @@ TEST(DriverInfoVKTest, DriverParsingArm) {
...
+  EXPECT_TRUE(IsBadVersionTest("Adreno (TM) 610"));
+  EXPECT_TRUE(IsBadVersionTest("Adreno (TM) 530"));
+  EXPECT_TRUE(IsBadVersionTest("Adreno (TM) 512"));
+  EXPECT_TRUE(IsBadVersionTest("Adreno (TM) 509"));
+  EXPECT_TRUE(IsBadVersionTest("Adreno (TM) 508"));
+  EXPECT_TRUE(IsBadVersionTest("Adreno (TM) 506"));
+  EXPECT_TRUE(IsBadVersionTest("Adreno (TM) 505"));
+  EXPECT_TRUE(IsBadVersionTest("Adreno (TM) 504"));
+
+  EXPECT_FALSE(IsBadVersionTest("Adreno (TM) 640"));
```
Comment (zanderso) [inline impeller/renderer/backend/vulkan/driver_info_vk_unittests.cc:93]: Looks like 640 needs to go in the `EXPECT_TRUE` list.

Comment (jonahwilliams) [reply impeller/renderer/backend/vulkan/driver_info_vk_unittests.cc]: oops, good catch

**Comment by sarope** [comment]
This is not yet on the current beta branch (3.28.0-0.1.pre). Could we get this cherry-picked there too?

**Pull request merged**